pdg/src/main.rs

```diff
@@ -191,6 +182,10 @@ fn main() -> eyre::Result<()> {
     pdg.graphs.assert_all_tests();
     let repr = pdg.repr(&args.print);
     println!("{repr}");
+
+    let f = std::fs::File::create("pdg.bc")?;
```

Comment:
This can be added as a CLI arg in `Args` pretty easily, something like `--output` maybe.

Reply:
Done in a78e49693fc02f3e70e9972f1933001683f2bce0.